client/src/app/pages/applications/components/application-analysis-status.tsx

```diff
@@ -34,3 +34,30 @@ export const ApplicationAnalysisStatus: React.FC<
 > = ({ state }) => {
   return <IconedStatus preset={getTaskStatus(state)} />;
 };
+export const mapAnalysisStateToLabel = (value: TaskState) => {
+  const presetKey: IconedStatusPreset = getTaskStatus(value);
+  const { t } = useTranslation();
```

Comment:
Unfortunately we cannot use the translation hook here as it's an ordinary method -  check  [Rules of Hooks](https://react.dev/warnings/invalid-hook-call-warning).
We could pass the `t` function as a param or build the presets outside.

Reply:
Done in https://github.com/konveyor/tackle2-ui/pull/2152/commits/d2096ca78fa99877f5a7eed32db9402a4e3c83eb